packages/integration-tests/api/distribution.js

```diff
@@ -55,6 +57,41 @@ async function getDistributionApiFileStream(fileUrl, accessToken) {
   return got.stream(s3SignedUrl);
 }
 
+
+/**
+ * Invoke distribution api lambda directly to get a signed s3 URL.  This is
+ * used in integration testing to we can be sure to use the lambda's iams
+ * permisions to generate correct access.
+ *
+ * @param {string} fileUrl
+ *   Distribution API file URL to request
+ * @param {string} accessToken
+ *   Access token from OAuth provider
+ * @returns {string}
+ *   signed s3 URL for the requested file.
+ */
+async function invokeLambdaForS3SignedUrl(fileUrl, accessToken) {
```

Comment:
😍  This is so awesome

Reply:
Well, it was a bit of trial and error and digging into the express code to see how it should be called, but it's nice for sure.